CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ and `Blankevoort1991Ligament`, usages of `get_GeometryPath`, `upd_GeometryPath`,
 - Simbody was updated such that the headers it transitively exposes to downstream projects are compatible with C++20 (#3619)
 - Moved speed computation from `computeForce` in children of `ScalarActuator` to dedicated `getSpeed` function.
 - Fix type problem with BufferedOrientationReference (Issue #3415, PR #3644)
+- Fixed setting the color of a PathSpring's GeometryPath should now update the color of the PathSpring geometry
```

Comment:
Minor, but this fix will end up in the next release (4.6) since we've recently finalized 4.5.

Reply:
If you do want this in 4.5, we'd have to merge it into branch `opensim_45` soon. Let @aymanhab know if that is the case.

Reply:
Whatever works: OSC is using main branch releases of OS these days, so that I can integrate/test API changes as they happen, rather than in spikes, so it isn't as important that  fixes are rolled into point releases for me

Reply:
Sounds good! Just wanted to make sure you were aware.